MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,9 +1,140 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
+    
+    // MARK: - Variables
+    
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var counterLabel: UILabel!
+
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+    
     // MARK: - Lifecycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        let currentQuestion = questions[currentQuestionIndex]
+        let currentViewModel = convert(model: currentQuestion)
+        show(quiz: currentViewModel)
+        
+    }
+    
+    // MARK: - Functions
+    
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        showAnswerResult(isCorrect: false)
+    }
+    
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        showAnswerResult(isCorrect: true)
+    }
+    
+    private func convert(model: QuizQuestion) -> QuizStepViewModel {
+        return QuizStepViewModel(
+            image: UIImage(named: model.image) ?? UIImage(),
+            question: model.text,
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)")
+        
+    }
+    
+    private func show(quiz step: QuizStepViewModel) {
+        // здесь мы заполняем нашу картинку, текст и счётчик данными
+        imageView.image = step.image
+        textLabel.text = step.question
+        counterLabel.text = step.questionNumber
+        
+    }
+    
+    private func show(quiz result: QuizResultsViewModel) {
+        // здесь мы показываем результат прохождения квиза
+        let alert = UIAlertController(title: result.title, message: result.text, preferredStyle: .alert)
+        let action = UIAlertAction(title: result.buttonText, style: .default, handler: {_ in self.currentQuestionIndex = 0
+            let firstQuestion = self.questions[self.currentQuestionIndex]
+            let viewModel = self.convert(model: firstQuestion)
+            self.show(quiz: viewModel)
+        })
```

Comment:
после in нужен перенос строки
```suggestion
        let action = UIAlertAction(title: result.buttonText, style: .default, handler: { _ in 
            self.currentQuestionIndex = 0
            let firstQuestion = self.questions[self.currentQuestionIndex]
            let viewModel = self.convert(model: firstQuestion)
            self.show(quiz: viewModel)
        })
```